addons/account/models/partner.py

```diff
@@ -285,6 +285,16 @@ class ResPartner(models.Model):
     _name = 'res.partner'
     _inherit = 'res.partner'
 
+    credit_limit = fields.Monetary(string="Credit Limit", compute='_compute_credit_limit', store=True, readonly=False)
```

Comment:
should be company dependent

Reply:
and store=False (because company dependent
@qdp-odoo

Reply:
Good point, but how would you make it company dependent when partners are not linked to a company ?
And if the field is not stored, how can it be set at the partner level (overriding the value set on the company) ?